readme.md

```diff
@@ -72,5 +103,42 @@ The data is based on https://www.kaggle.com/datasets/wolfram77/graphs-snap-soc-p
 licensed: https://creativecommons.org/licenses/by/4.0/
 
 
+## FAQ
+
+### System Requirements
+**Q: What are the minimum system requirements?**  
+A: FalkorDB requires a Linux/Unix system with 4GB RAM minimum. For production environments, 16GB RAM is recommended.
+
+### Installation & Setup
+**Q: Can I run FalkorDB without Redis?**  
+A: No, FalkorDB requires Redis 6.2 or higher as it operates as a Redis module.
+
+### Development
+**Q: Which query language does FalkorDB use?**  
+A: FalkorDB uses the Cypher query language, similar to Neo4j, making migration straightforward.
+
+### Data Management
+**Q: Does FalkorDB support data persistence?**  
+A: Yes, through Redis persistence mechanisms (RDB/AOF). Additional persistence options are in development.
+
+### Integration
+**Q: Does FalkorDB support common programming languages?**  
+A: Yes, through Redis clients and the Bolt protocol, supporting major languages including Python, Java, Go, and Node.js.
```

Comment:
@barakb FalkorDB has a set of Clients in all these programming languages and more (e.g. Rust)

You should add a link to the clients page on our docs